### DISCUSSION TITLE
Suggestion: "Mean field" histidine

### DISCUSSION THREAD

**Discussion by KasperBuskPedersen** [marrink-lab/vermouth-martinize#507] [Announcements]
Hi @fgrunewald and martinize2 developers (even though this issue would probably be more appropriate in a protein development group)

Histidine pka can be significantly shifted inside proteins, often in the ranges of 5-7, sometimes even higher, meaning that at pH 7 one histidine have significant subpopulations of both the neutral tautomers and the protonated charged histidine. 

In a coarse-grained model it would therefore make good sense to have a mean field topology (in lack of a better word), halfway between say HIE and HIP. Something like:

P2             HIM BB  0.0
TC4            HIM SC1 0.0
TN6d           HIM SC2 0.25
TN6d           HIM SC3 0.25

Is this something you have explored? The issue is ofc that the non-integer charge should be counteracted by some small number of modified ions e.g.
NAX charge=0.9

Best regards
Kasper

**Comment by fgrunewald** [comment]
@KasperBuskPedersen we have not explore a mean-field histidine. Of course a constant pH approach would really be the better solution, which is something we work on. However, I'm not sure given the parametrisation strategy of Martini, that a mean-field HIS residue would yield an improved accuracy over chosen the more prevalent form. Do you have any specific data or test-case?

FYI I will convert this issue to a discussion, given that it concerns the protein model on a more general level rather than a code issue.

**Comment by KasperBuskPedersen** [comment]
Hi @fgrunewald 

Thanks :)

I do not have a good test case in mind sorry - apart from the fact that i have to deal with such a histidine in my own application system. It was more a general note that one could imagine a CG histidine model that could describe multiple tautomers - though im not sure it is a good idea to even make one if constant pH is being implemented - constant pH would be much more appropiate. 

Cheers
Kasper
